apps/wing-console/console/ui/src/layout/vscode-layout.tsx

```diff
@@ -89,84 +89,88 @@ export const VscodeLayout = ({ cloudAppState, wingVersion }: LayoutProps) => {
       >
         <Header title={wingfile.data ?? ""} />
         <div className="flex-1 flex relative">
-          {loading && (
-            <div
-              className={classNames(
-                "absolute h-full w-full z-50 bg-white/70 dark:bg-slate-600/70",
-              )}
-              data-testid="loading-overlay"
-            >
-              <div className=" absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2">
-                <SpinnerLoader />
-              </div>
-            </div>
+          {cloudAppState === "error" && (
+            <BlueScreenOfDeath
+              title={"An error has occurred:"}
+              error={errorMessage.data ?? ""}
+            />
```

Comment:
Can we render only this when there's an error? There's no need to keep the other elements. For example:

```ts
if (cloudAppState === "error" && errorMessage.data) {
  return <BlueScreenOfDeath ... />;
}

return (
    <>
      {showTerms && (
        <TermsAndConditionsModal ...
```

Reply:
We also show the header and footer when the blue screen is displayed.